clusterloader2/testing/load/config.yaml

```diff
@@ -27,9 +27,9 @@
 name: load
 automanagedNamespaces: {{$namespaces}}
 tuningSets:
-- name: Uniform100qps
-  qpsLoad:
-    qps: 100
+- name: Sequence
+  parallelismLimitedLoad:
+    parallelismLimit: 1
```

Comment:
We can probably increase it to more than one, but I'm fine with doing that later.